terms.html

```diff
@@ -7,6 +7,13 @@
 
 <dl class="termlist">
 
+  <dt><dfn>authentication</dfn></dt>
+  <dd>
+Authentication is a mechanism by which an entity can prove it is the <a>DID
+subject</a>, using one or more of a set of verification methods (such as, but
+not limited to, public keys).
+  </dd>
+
```

Comment:
We need to improve this definition, it's inaccurate... /cc @kdenhartog @talltree @dlongley